co-noir/co-brillig/src/brillig_vm.rs

```diff
@@ -315,7 +342,7 @@ where
                     let result = self.driver.cmux(condition.clone(), truthy, falsy)?;
                     final_result.push(result);
                 }
-                Ok(Some(CoBrilligResult::Success(final_result)))
+                Ok(Some(CoBrilligResult::Success(final_result, HashMap::new())))
```

Comment:
this does not seem right? can the PSS not have some values before the if and would they not get dropped here?